internal/pkg/vm/vm.go

```diff
@@ -36,7 +36,7 @@ func send(ctx context.Context, deps sendDeps, vmCtx *Context) ([][]byte, uint8,
 		}
 	}
 
-	if vmCtx.message.Method == "" {
+	if vmCtx.message.Method == types.InvalidMethodID {
```

Comment:
No method is a valid method ID. This should be something like `types.NoMethodID`.